### DISCUSSION TITLE
General Priiloader Testing and Hack Creation Discussion

### DISCUSSION THREAD

**Discussion by Ingunar** [DacoTaco/priiloader#354] [General]
<p align="center" width="100%">
  <img width="720" alt="fidget spinner" src="https://raw.githubusercontent.com/DacoTaco/priiloader/master/Images/Logo.png">
</p>

<h2> <b>Everyone is welcome to help with testing, porting, and creation of new hacks!</b></h2>

**You can find more info about the hacks and testing progress in the note:**
[**Priiloader Hacks - Testing and Info**](https://hackmd.io/@Ingunar/PriiloaderTesting)

**You can read more about the hack creation here:**
[**About Priiloader Hacks**](https://github.com/DacoTaco/priiloader/blob/master/docs/HACKS.md)

**You can read information about the current hacks here:**
[**List of Priiloader Hacks**](https://github.com/DacoTaco/priiloader/blob/master/docs/HACKSLIST.md)

**Frequently Asked Questions regarding Priiloader:**
[**Priiloader FAQ**](https://github.com/DacoTaco/priiloader/blob/master/docs/FAQ.md)

**Comment by Ingunar** [comment]
## Reserved

**Comment by Ingunar** [comment]
## **New stable `hacks_hash.ini` was added to the release page!**
**Permanent vWii System Settings and Wiimmfi Patch v4 for vWii; Many corrections, improvements, and new hacks for Wii and Mini**
### [Download the Latest Version here](https://github.com/DacoTaco/priiloader/blob/master/src/priiloader/hacks_hash.ini) and replace your old one.


*When updating  `hacks_hash.ini` you don't need to reinstall Priiloader, but you will need to re-enable the hacks again.*

**Version 0.10.1 - RELEASE**

Changes:
- ADDED [Remove Pillarboxing Completely] for vWii
- ADDED [Remove Pillarboxing from VC Titles] for vWii
- ADDED [Permanent vWii System Settings] for vWii<br>(Please read [**How to use Permanent Wii System Settings on vWii**](https://dacotaco.github.io/priiloader/docs/FAQ.html#how-to-use-permanent-wii-system-settings-on-vwii))
- RENAMED [Always enable WiiConnect24 for vWii] to [Always enable WiiConnect24 on boot]
- ADDED [Force Disc Games to run under IOS248] for Wii
- ADDED [Force Disc Games to run under IOS249] for Wii Mini
- ADDED [Force Disc Games to run under IOS250] for Wii and vWii
- ADDED [Force Disc Games to run under IOS251] for Wii and vWii
- ADDED [Wiimmfi Patch v4] for vWii
- REMOVED [No System Menu Sounds AT ALL]
- ADDED [No System Menu SFX & Background Music] for Wii, vWii and Mini
- RENAMED [Allow TitleID RAAE,408x,410x] to [Unblock Service Discs]
- ADDED [Unblock Service Discs] for all of the System Menu versions
- ADDED [Region Free EVERYTHING] for USA Wii Mini
- ADDED [Block Disc Autoboot] and [Remove IOS16 Disc Error] for vWii and Wii Mini
- ADDED [Create message via Calendar button] for Wii Mini
- Many, many, MANY code corrections

<details>
  <summary> <b> Show all previous versions  </b> </summary>
<p>

Version [0.10.1 - v03](https://cdn.discordapp.com/attachments/1078641159510958090/1159938944083963954/hacks_hash.ini)
- ADDED [Wiimmfi Patch v4 for vWii 5.2.0x] for vWii

Version: [0.10.1 - v02](https://cdn.discordapp.com/attachments/1078641159510958090/1154207070984355840/hacks_hash.ini)

- ADDED [Permanent vWii System Settings] for vWii<br>Please read **Explanation about the Permanent vWii System Settings** below.
- REMOVED [Disable SYSCONF overwrite on boot]<br>(It was replaced with the [Permanent vWii System Settings])
- ADDED [Force Disc Games to run under IOS248] for Wii
- ADDED [Force Disc Games to run under IOS250] for Wii and vWii
- ADDED [Force Disc Games to run under IOS251] for Wii and vWii
- ADDED [No System Menu SFX & Background Music] for Wii, vWii and Mini

Version [0.10.1 - v01](https://cdn.discordapp.com/attachments/1078641159510958090/1152931154861764658/hacks_hash.ini)

- REMOVED [No System Menu Sounds AT ALL]
- RENAMED [Always enable WiiConnect24 for vWii] to [Always enable WiiConnect24 on boot]
- RENAMED [Allow TitleID RAAE,408x,410x] to [Unblock Service Discs]
- ADDED [Unblock Service Discs] for all of the System Menu versions
- ADDED [Region Free EVERYTHING] for USA Wii Mini
- ADDED [Force Disc Games to run under IOS249] for Wii Mini
- ADDED [Block Disc Autoboot] and [Remove IOS16 Disc Error] for vWii and Wii Mini
- ADDED [Create message via Calendar button] for Wii Mini
- Verified and fixed min and max versions for all of the hacks
- Fixed the order of some hacks

Version [0.10.0 - Test 01b](https://cdn.discordapp.com/attachments/1078641159510958090/1150094894837268520/hacks_hash.ini)
*(Re-uploaded as "Test 01b" with a fix that caused an invalid line)*
- REMOVED incompatible hacks for Wii Mini - [Remove NoCopy Save File Protection], [Re-Enable Bannerbomb v2], [Mark Network Connection as Tested] - SD and WiFi cards mods require 51x version of System Menu

Version [0.10.0 RC3 - Test 02](https://cdn.discordapp.com/attachments/1078641159510958090/1139984956513341440/hacks_hash.ini)
- FIX [No System Menu Background Music] - Wii, Wii Mini

Version [0.10.0 RC3 - Test 01](https://cdn.discordapp.com/attachments/1078641159510958090/1130974736046882866/hacks_hash.ini)
- Re-enabled [Disable SYSCONF overwrite on boot] for vWii
- Re-enabled [Remove IOS16 Disc Error] for vWii
- Re-enabled [Allow TitleID RAAE,408x,410x] for vWii
- Re-enabled [Block Disc Autoboot] for vWii
- Re-enabled [No System Menu Sounds AT ALL] for all

Version [0.10.0 RC3](https://github.com/DacoTaco/priiloader/blob/master/src/priiloader/hacks_hash.ini)
- DISABLED for vWii - [Block Disc Autoboot] , [Allow TitleID RAAE,408x,410x] , [Remove IOS16 Disc Error] - No use for them currently
- DISABLED for all System Menu versions - [No System Menu Sounds AT ALL] Not known to be working properly with any Wii SM version
- FIX [Remove NoCopy Save File Protection] - vWii JAP and vWii US
- FIX [No System Menu Background Music] - vWii JAP
- REMOVED [Fix NWC24iSetUniversalTime (UTC+1)] - It will need to be generated and added manually from now on
- Added comments for the disabled hacks

Version [09.07.23](https://cdn.discordapp.com/attachments/1078641159510958090/1127654018156605500/hacks_hash.ini)

- Removed most of the comments
- Name change [Enable Wii System Settings for vWii] to [Wii System Settings via Options Button]
- Name change [Message board write for vWii] to [Create message via Calendar button]
- Old [Remove Deflicker] hack removed
- Name change [NEW Remove Deflicker] to [Remove Deflicker]
- NEW HACK [Fix NWC24iSetUniversalTime (UTC+1)] 
(Experimental vWii timestamp fix, created only for the PAL and UTC+1 ;
 Disabled by default)

Version [11.05.23](https://cdn.discordapp.com/attachments/1078641159510958090/1106194787138289735/hacks_hash.ini)

- Old [Remove Deflicker] is renamed to [OLD Remove Deflicker] and disabled
- NEW HACK [Disable SYSCONF overwrite on boot] was added but disabled (since it can currently cause a conflict with [Always enable WiiConnect24 for vWii] hack)

Version [05.03.23](https://cdn.discordapp.com/attachments/1078641159510958090/1082003724987269160/hacks_hash.ini)
- NEW CODE [Replace Health Screen with Backmenu] - Added support for 2.x
- NEW HACK [Skip Health Screen] - [Replace Health Screen with Backmenu] alternative for SM 1.0 Only. 
Warmboot on 1.0 just skips the Health Screen, there's no Backmenu animation for it to use instead

Version [04.03.23](https://cdn.discordapp.com/attachments/1078641159510958090/1081566124279341066/hacks_hash.ini)
- FIX [Always enable WiiConnect24 for vWii] for JPN
- FIX [Force Boot into Data Management] for 3.x
- [NEW Remove Deflicker] - changed the minversion

Version [03.03.23](https://cdn.discordapp.com/attachments/1078641159510958090/1081198047985405993/hacks_hash.ini)
- NEW CODE for [Force Boot into Data Management] - added support for more versions
- [480p graphics fix in system menu] - minversion was lowered to remove vWii and Wii Mini from the list

Version [02.03.23b](https://cdn.discordapp.com/attachments/1078641159510958090/1080845391039303741/hacks_hash.ini)
- Name change [New Remove Deflicker by Vague Rant] is now called [NEW Remove Deflicker]
- NEW CODE for [Replace Health Screen with Backmenu], one hack for all of the versions
- Old [Replace Health Screen with Backmenu] codes removed
- NEW CODE [Region Free EVERYTHING] for JAP vWii added

Version [02.03.23](https://cdn.discordapp.com/attachments/1078641159510958090/1080704847369732136/hacks_hash.ini)
- [Block Disc Updates] was split for Wii and Wii-mini, and set to ignore vWii
- NEW CODE [Remove NoCopy Save File Protection] for vWii added. The old master was not working well with vWii so this had to be rewritten
- NEW CODE [Region Free EVERYTHING] for vWii added
- NEW CODE [No System Menu Background Music] added
- NEW HACK [Force Boot into Data Management] for vWii and Wii added
- NEW CODE [Remove Diagnostic Disc Check] for vWii added
- [Force Standard Recovery Mode] - upped maxversion to support vWii
- [Force Disc Games to run under IOS249] - upped maxversion to support vWii
- [Remove Deflicker] - upped maxversion to support vWii
- NEW CODE/HACK [New Remove Deflicker by Vague Rant] added
- Name change [Mark Network Con as Tested] is now called [Mark Network Connection as Tested]
- [Mark Network Connection as Tested] was enabled
- NEW HACK [Always enable WiiConnect24 for vWii] added
- NEW HACK [Enable Wii System Settings for vWii] added
- NEW HACK [Message board write for vWii] added

</p>
</details>

**Reply by Armandprogame** [reply]
Adinaton, do you know since Leseratte when the wiimmfi v4 priiloader patch for vwii will be released?

**Reply by Ingunar** [reply]
I am not sure what you are asking, but for anything related to Wiimmfi v4 Patch for Priiloader on vWii you need to ask @Leseratte10

**Reply by Armandprogame** [reply]
How do I create a priiloader patch, especially the one for wiimmfi

**Reply by Leseratte10** [reply]
Patch is now available at https://github.com/DacoTaco/priiloader/pull/358 and can be tested.

**Reply by Armandprogame** [reply]
You did it Leseratte

**Reply by Armandprogame** [reply]
the only thing missing is to unify the wii u competition patch with the wii one

**Reply by Armandprogame** [reply]
Leseratte but can you give me the invitation to the official English wiimmfi server which I can't find on wii homebrew?

**Reply by fffoo** [reply]
i thought it was just me that had something wrong with online updates and now i know it was the goddamn host

**Reply by Armandprogame** [reply]
Hi adinaton but is the update of the competition patcher for vwii finished?

**Reply by DacoTaco** [reply]
competition patcher for vwii? is this related to wiimmfi?

**Reply by Armandprogame** [reply]
no I mean the wiimmfi competitions patch for wii u have you unified it with the wii one?

**Reply by Armandprogame** [reply]
and if so, can you publish it?

**Reply by DacoTaco** [reply]
i have not gotten any new patch from wiimmfi, and if there is a new one i suggest you talk to them to get it published..

**Reply by Armandprogame** [reply]
But adinaton (ingunar) he receive it

**Reply by Ingunar** [reply]
Please stop spamming! And please find something better to do. 
Do not bother developers about it anymore. Your comments regarding this will be deleted.

It will be done when it is ready, and it will be shared [HERE](https://forum.wii-homebrew.com/index.php/Thread/55629-MKWii-Competitions/)

**Comment by steamywontons** [comment]
I used a normal Wii motion plus inside controller.Sent from my iPhoneOn Nov 26, 2023, at 10:15 AM, Ingunar ***@***.***> wrote:﻿

What controller are you using?
(Wiimote or GC)
If it is Wiimote, what model is it?

**Reply by DacoTaco** [reply]
what do you mean with the controller disconnects? when does it disconnect? what if you press A on the wiimote after it disconnecting?

**Comment by Vinnievovi** [comment]
I have 2 themes on vWii, default and black, which I can install with mymenuify.  If I install priiloader and decide to change theme, will it brick?

**Reply by DacoTaco** [reply]
as is, yes it will brick vwii (which can be fixed from the wiiu side). its better to first uninstall priiloader, install the theme and then install priiloader again. there are new vwii theme installers in the works that do things the right way, which won't brick vwii
